src/frequency.c

```diff
@@ -48,7 +43,7 @@ int32_t frequency_get_error()
     if (!frequency) {
         return 0;
     } else {
-        int32_t error = frequency - HAL_RCC_GetHCLKFreq();
+        int32_t error = frequency - 70000000 /*HAL_RCC_GetHCLKFreq()*/;
```

Comment:
What's the reason for not using HAL_RCC_GetHCLKFreq? I usually prefer not to have magic numbers in the code.

Reply:
Hi Daniel,
Just trying to get the code to execute faster for timing issues.
Here we avoid a function call with context switching and pushing values to the stack etc.